services/httpd/response_logger.go

```diff
@@ -84,11 +85,19 @@ func buildLogLine(l *responseLogger, r *http.Request, start time.Time) string {
 	username := parseUsername(r)
 
 	host, _, err := net.SplitHostPort(r.RemoteAddr)
-
 	if err != nil {
 		host = r.RemoteAddr
 	}
 
+	if xff := r.Header["X-Forwarded-For"]; xff != nil {
+		addrs := make([]string, 0, len(xff)+1)
```

Comment:
Instead of:

```go
addrs := make([]string, 0, len(xff)+1)
for _, addr := range xff {
	addrs = append(addrs, addr)
}
addrs = append(addrs, host)
```

just do: 

```go
addrs := append(xff, host)
```